main.go

```diff
@@ -49,6 +49,7 @@ type GlobalFlags struct {
 	LogLevel              string   `name:"log.level" help:"Only log messages with the given severity or above. Valid levels: [debug, info, warn, error, fatal]" enum:"debug,info,warn,error,fatal" default:"error"`
 	ConnectTimeoutMS      int      `name:"mongodb.connect-timeout-ms" help:"Connection timeout in milliseconds" default:"5000"`
 
+	DisableExporterMetrics   bool `name:"web.disable-exporter-metrics" help:"Exclude metrics about the exporter itself (process_*, go_*)"`
```

Comment:
I believe it should be `--no-collector.exporter-metrics`

Reply:
Yeah this is probably more in line with the naming the exporter uses. I was not sure which name is preferred so i was taking inspiration from the node_exporter. I will change the name and update the PR.